### ISSUE TITLE
noise problem

### ISSUE THREAD

**Issue by FourierTransform** [xinntao/EDVR#122]
I use EDVR on my dataset(there are some videos with noise,others are clean).I have performed 160k times,but the test curve doesn't look normal,
![image](https://user-images.githubusercontent.com/31101482/69492532-ff7afc00-0ede-11ea-8bed-541c33ace97f.png)
![image](https://user-images.githubusercontent.com/31101482/69492540-16b9e980-0edf-11ea-9ad1-8fc7aca562e8.png)
Though after 160k iterations,it still can't work well on noise image.I wonder know the reason

**Comment by xinntao** [comment]
yes, the curve looks strange - it seems that the model overfits the training data. 
Do your test video contains different noise types from the training videos?

**Comment by FourierTransform** [comment]
the noise are same
